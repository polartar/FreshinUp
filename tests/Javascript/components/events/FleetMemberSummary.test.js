import { createLocalVue, mount } from '@vue/test-utils'
import Component from '~/components/events/FleetMemberSummary.vue'
import { FIXTURE_EVENT_FLEET_MEMBER_SUMMARY } from 'tests/__data__/fleetMemberSummary'

describe('Event FleetSummary component', () => {
  // Component instance "under test"
  let localVue
  describe('Snapshots', () => {
    test('required set', () => {
      localVue = createLocalVue()
      const wrapper = mount(Component, {
        localVue: localVue,
        propsData: {
          member: FIXTURE_EVENT_FLEET_MEMBER_SUMMARY
        }
      })
      expect(wrapper.element).toMatchSnapshot()
    })
  })
})
