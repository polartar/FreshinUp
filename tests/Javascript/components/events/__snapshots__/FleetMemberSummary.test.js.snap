// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Event FleetSummary component Snapshots required set 1`] = `
<div
  class="layout"
>
  <div
    class="flex md4"
  >
    <div
      class="v-card v-sheet theme--light"
    >
      <div
        class="container header grey--text"
      >
        
        Fleet Member Summary
      
      </div>
       
      <hr
        class="v-divider theme--light"
      />
       
      <div
        class="container pt-0"
      >
        <div
          class="layout row align-center"
        >
          <div
            class="flex xs6 title grey--text"
          >
            <div
              class="v-card__text"
            >
              owner
            </div>
          </div>
           
          <div
            class="flex xs6 text-right grey--text"
          >
            <div
              class="v-card__text"
            >
              Dan Smith
            </div>
          </div>
        </div>
         
        <hr
          class="v-divider theme--light"
        />
         
        <div
          class="layout row align-center"
        >
          <div
            class="flex xs6 title grey--text"
          >
            <div
              class="v-card__text"
            >
              lisence due
            </div>
          </div>
           
          <div
            class="flex xs6 text-right grey--text"
          >
            <div
              class="v-card__text"
            >
              Dec, 30 2020
            </div>
          </div>
        </div>
         
        <hr
          class="v-divider theme--light"
        />
         
        <div
          class="layout row align-center"
        >
          <div
            class="flex xs6 title grey--text"
          >
            <div
              class="v-card__text"
            >
              contact phone
            </div>
          </div>
           
          <div
            class="flex xs6 text-right grey--text"
          >
            <div
              class="v-card__text"
            >
              938 374822
            </div>
          </div>
        </div>
         
        <hr
          class="v-divider theme--light"
        />
         
        <div
          class="layout row align-center"
        >
          <div
            class="flex xs6 title grey--text"
          >
            <div
              class="v-card__text"
            >
              contact email
            </div>
          </div>
           
          <div
            class="flex xs6 text-right grey--text"
          >
            <div
              class="v-card__text"
            >
              dan.simth@gmail.com
            </div>
          </div>
        </div>
         
        <hr
          class="v-divider theme--light"
        />
         
        <div
          class="layout xs6 title grey--text mt-2"
        >
          <div
            class="v-card__text"
          >
            TAGS
          </div>
        </div>
         
        <div
          class="layout row wrap"
        >
          <span
            class="v-chip theme--light primary primary-text--text"
            tabindex="0"
          >
            <span
              class="v-chip__content"
            >
              
            SEAFOOD
          
            </span>
          </span>
          <span
            class="v-chip theme--light primary primary-text--text"
            tabindex="0"
          >
            <span
              class="v-chip__content"
            >
              
            SMOKED
          
            </span>
          </span>
          <span
            class="v-chip theme--light primary primary-text--text"
            tabindex="0"
          >
            <span
              class="v-chip__content"
            >
              
            DESSERT
          
            </span>
          </span>
          <span
            class="v-chip theme--light primary primary-text--text"
            tabindex="0"
          >
            <span
              class="v-chip__content"
            >
              
            BAY AREA
          
            </span>
          </span>
          <span
            class="v-chip theme--light primary primary-text--text"
            tabindex="0"
          >
            <span
              class="v-chip__content"
            >
              
            VEGAN OPTIONS
          
            </span>
          </span>
          <span
            class="v-chip theme--light primary primary-text--text"
            tabindex="0"
          >
            <span
              class="v-chip__content"
            >
              
            SMOKED
          
            </span>
          </span>
          <span
            class="v-chip theme--light primary primary-text--text"
            tabindex="0"
          >
            <span
              class="v-chip__content"
            >
              
            DESSERT
          
            </span>
          </span>
          <span
            class="v-chip theme--light primary primary-text--text"
            tabindex="0"
          >
            <span
              class="v-chip__content"
            >
              
            SEAFOOD
          
            </span>
          </span>
          <span
            class="v-chip theme--light primary primary-text--text"
            tabindex="0"
          >
            <span
              class="v-chip__content"
            >
              
            SMOKED
          
            </span>
          </span>
        </div>
         
        <hr
          class="mt-3 v-divider theme--light"
        />
         
        <button
          class="button-grey grey mt-4 v-btn theme--light"
          type="button"
        >
          <div
            class="v-btn__content"
          >
            
          View Fleet Member Profile
        
          </div>
        </button>
         
        <button
          class="button-remove grey--text mt-3 v-btn theme--light"
          type="button"
        >
          <div
            class="v-btn__content"
          >
            <i
              aria-hidden="true"
              class="v-icon icon left far fa-trash-alt theme--light"
            />
             
            <v-text>
              Remove fleet member from this event
            </v-text>
          </div>
        </button>
      </div>
    </div>
  </div>
</div>
`;
